ui/app/styles/components/box-label.scss

```diff
@@ -10,7 +10,6 @@ label.box-label {
 .box-label {
   @extend .box;
   @extend .is-centered;
-  @extend .is-gapless;
```

Comment:
`is-gapless` was a bulma style that could only be used with columns (e.g `columns.is-gapless`). See [bulma docs](https://bulma.io/documentation/columns/gap/#gapless). Once I moved the class out of bulma-classes this threw an error. I double checked that this still styled okay without the extend.